src/components/notes/list.styl

```diff
@@ -0,0 +1,27 @@
+.empty { 
```

Comment:
[Our guidelines](https://docs.cozy.io/cozy-ui/styleguide/) say we don't use `{}`